src/views/Header/HeaderBackButton.js

```diff
@@ -97,7 +97,7 @@ class HeaderBackButton extends React.PureComponent<DefaultProps, Props, State> {
             source={asset}
           />
           {Platform.OS === 'ios' &&
-            title &&
+            typeof title === 'string' &&
```

Comment:
```js
typeof '' === 'string' // true
```

This will still render the `<Text>` when `title === ''`.

Reply:
i think this behavior is correct, and more perfomant on blinking

Reply:
Sorry for the delay on this-- looking back at this again, would it not be better to check `typeof backButtonTitle` since that is what is being rendered?